pyiron/atomistics/structure/atoms.py

```diff
@@ -1937,6 +1937,27 @@ def get_voronoi_volume(self):
         )
         return self.analyse_ovito_voronoi_volume()
 
+    def find_mic(self, v, vectors=True):
+        """
+        Find vectors following minimum image convention (mic). In principle this
```

Comment:
I am ok accepting this as is, but putting myself in the shoes of a new MSc student I see that it might be useful to include a very brief description of what "minimum image convention" is.